docs/data-sources/kubernetes_secret.md

```diff
@@ -0,0 +1,87 @@
+---
+Title: "Secret Resource"
+Description: |-
+    Fetching the details of sceret resource.
+---
+
+# Namespace
```

Comment:
Please correct the title- Secret/ Kubernetes Secret

Reply:
updated PTAL